Сетка страниц и крупных блоков на флексах

KuzminaPolina opened the pull request:
Домашка по блоку "5.21. Сетка страниц и крупных блоков на флексах"

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/35/user/2098917/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2098917-device-35/5/

KuzminaPolina committed:
adding pages' basic structure

KuzminaPolina committed:
slider layout and images correction

KuzminaPolina committed:
adding back images

KuzminaPolina committed:
making adjustments to slider

KuzminaPolina committed:
deleted a comment

senchkim commented on catalog.html:
```diff
@@ -16,77 +16,83 @@
       <header class="page-header">
         <h1 class="visually-hidden hidden-my-way">Интернет-магазин "Девайс"</h1>
 
```
`01.svg` - названия плохие у картинок! (не желательно чтобы название начиналось с цифры!)
Назови лучше `number-one.svg` 👍 

Исправить все 3 названия

senchkim commented on index.html:
```diff
@@ -16,7 +16,7 @@
       <h1 class="visually-hidden hidden-my-way">Интернет-магазин "Девайс"</h1>
 
-        <div class="transparent-header-menu-wrapper">
+        <div class="transparent-header-menu-wrapper-index">
```
https://validator.w3.org/nu/#textarea
Ошибки в валидаторе, причем на обоих страницах!

senchkim commented on styles/style.css:
```diff
@@ -38,15 +42,26 @@ body {
-.hidden-my-way {
-  text-indent: -9999px;
+/*.visually-hidden */
+.hidden-my-way{
```
Класс должен так называться `.visually-hidden`

KuzminaPolina committed:
images fix

KuzminaPolina committed:
images fix

KuzminaPolina committed:
fixes

1. Images names;
2. Validator mistakes, except icons' height in long button;
3. Visually hidden class;
